conjur/init_controller.py

```diff
@@ -0,0 +1,77 @@
+# Builtin
+import logging
+import os
+import sys
+
+from urllib.parse import urlparse
+
+class InitController:
+    """
+    InitController
+
+    This class represents the Presentation Layer for the init command
+    """
+    DEFAULT_CONFIG_FILE = os.path.expanduser(os.path.join('~', '.conjurrc'))
+
+    force_overwrite = False
+    init_input_obj = None
+    init_logic = None
+
+    def __init__(self, init_input_obj, init_logic, force):
+      self.init_input_obj = init_input_obj
+      self.init_logic = init_logic
+      self.force_overwrite = force
+
+    def call(self):
+      fetched_certificate = self.get_server_certificate(self.init_logic, self.init_input_obj)
```

Comment:
Do you want to catch and hendel the exception of ValueError("You decided not to trust the certificate")?

Reply:
wouldn't it just perc up to the main caller (cli.py)? There I have a try/catch for ValueError